### PR TITLE
french translation of round 7

### PR DESCRIPTION
**Pull request by rafasashi**
I have translated the round 7 in french. I rewrote the output given by AWS Neural Machine Translation but it was apparently not "Neural friendly" so it might require proofreading from another native speaker familiar with the technical terms before publishing.

### CODE REVIEW

**Comment by vbuterin** [comment]
Thanks a lot for this!

> les détenteurs d'un jeton détiennent également

I'm curious, is "jeton" normally used to refer to crypto tokens in the French crypto community? I've only ever encountered that word used for subway tokens! (hmm.... Google searching, within .fr sites, it looks like "un token ERC20" beats "un jeton ERC20" 282-204)

Would definitely be good to get some review from someone familiar with the technical terms; I'm sure there's crypto-space-specific nuances that machine transactions would not be very good at.

**Comment by rafasashi** [comment]
I agree. First thing, I am a crypto enthusiast not an active member of the French crypto community, I have a Coinbase account with some ETH in it and I follow many Crypto experts on twitter to keep me informed but I have no authority in the matter (I am saying this in case any French fellow is reading this). This being said the word “jeton” for “token” in the crypto context is acceptable because it is the one used by the french lawmakers. Check the word Token in wikipedia page here:

https://fr.wikipedia.org/wiki/Cryptomonnaie#Cryptomonnaie_et_titre_de_cr%C3%A9ance  

In Google the English word beats the French one because there are 2 schools when it comes to internationalization of knowledge in France. The one who hates mixing french with any form of foreign language, and the one who prefers to use the language that dominates one topic whenever the french translation does not sound “cool”, “professional” or simply “appropriate”. It is a weird thing to say, but it really depends on the social context.

In the case of Crypto currency there is a lot of both behaviors because it's very new and also part of both economic and computer sciences. Now that Cryptocurrencies are more popular, scholars and other mass media are trying to make people use Crypto-monnaie instead of Cryptocurrency. 

Same trend exists with Coins & Tokens (without making the difference) people mostly use the english words on social media and niche blogs but lawmakers use Monnaie & Jeton and by extension some politicians and part of the banking industry that has nothing to do with coding, mining or trading, might also use the french terms.  

The best approach for translating Latin languages for European readers would be to pick reviewers that represent your target audience. Do you want to reach or be quoted by policymakers or by the tech community?

**Comment by jdetychey** [comment]
We do indeed the word Jeton for Token and it's an appropriate translation. Also the French tends to translate everything as required from the law (it's mandatory for commercial advertisements and public announcements to be given in French https://en.wikipedia.org/wiki/Toubon_Law)

File: posts/round7_FR.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+[categorie] : <> (Général)
+[date] : <> (2020/10/18)
+[titre] : <> ( 7ème tour des subventions Gitcoin - Rétrospective )
+[pandoc] : <> (—mathjax)
+
+# 7ème tour des subventions Gitcoin - Rétrospective
+
+Le septième tour des subventions Gitcoin s'est terminé avec succès! Ce tour a connu une augmentation de l'intérêt pour Gitcoin sans précédent, ce qui s'est traduit par 274.830 $ de contributions et 450.000 $ de fonds de contrepartie répartis sur 857 projets.
+
+La structure des catégories a été une fois de plus modifiée; cette fois-ci, divisé entre "tech DApp", "infrastructure" et "communauté". Voici les résultats :
```
Comment (jdetychey) [inline posts/round7_FR.md:10]: `La structure des catégories a été une fois de plus modifiée; cette fois-ci, divisé entre "tech DApp", "infrastructure" et "communauté". Voici les résultats :`
La structure des catégories a été une fois de plus modifiée; cette fois-ci, divisée entre "tech DApp", "infrastructure" et "communauté". Voici les résultats :

Comment (jdetychey) [reply posts/round7_FR.md:10]: "divisé" takes the gender of "structure", juste as "modifié**e** "
=> **divisée**

File: posts/round7_FR.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+<center>
+<img src="/images/round_7_files/results1.png" style="width: 360px" />
+<img src="/images/round_7_files/results2.png" style="width: 360px" />
+<img src="/images/round_7_files/results3.png" style="width: 360px" />
+</center>
+
+## Defi s'associe au mouvement !
+
+Lors de ce tour, le nombre de projets associés s'est avéré plus élevées qu'auparavant. En plus des associations habituels fournis par la Fondation Ethereum ainsi que quelques autres acteurs, se sont ajoutés pour la première fois différents projets defi :
```
Comment (jdetychey) [inline posts/round7_FR.md:20]: `Lors de ce tour, le nombre de projets associés s'est avéré plus élevées qu'auparavant. En plus des associations habituels fournis par la Fondation Ethereum ainsi que quelques autres acteurs, se sont ajoutés pour la première fois différents projets defi :`
Lors de ce tour, le nombre de projets associés s'est avéré plus élevé qu'auparavant. En plus des thèmes habituels fournis par la Fondation Ethereum ainsi que par quelques autres acteurs, se sont ajoutés pour la première fois différents projets defi :

File: posts/round7_FR.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+ ![] (/images/round_7_files/matching.png)
+
+Les participants étaient :
+
+* [Chainlink] (https://chain.link/), un projet Oracle smart contract
+* [Optimisme] (https://optimism.io/), un Rollup optimiste de la couche 2
+* La [Fondation Ethereum] (http://ethereum.org)
+* [Balancer] (https://balancer.exchange/), une bourse décentralisé
```
Comment (jdetychey) [inline posts/round7_FR.md:29]: `une bourse décentralisé`
=>
une bourse décentralisée

File: posts/round7_FR.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+* [Synthetix] (https://synthetix.io/), une plateforme d'actifs synthétiques
+* [Yann] (https://yearn.finance/), une plateforme de prêt garanti
```
Comment (jdetychey) [inline posts/round7_FR.md:31]: `une plateforme de prêt garanti`

une plateforme de prêts garantis

**Pull request merged**